plasmapy/atomic/atomic.py

```diff
@@ -266,10 +271,10 @@ def isotope_symbol(argument, mass_numb=None):
 
         if mass_numb is not None and mass_numb_from_arg is not None:
             if mass_numb == mass_numb_from_arg:
-                raise UserWarning("Redundant mass number information in " +
-                                  "isotope_symbol from inputs: (" +
-                                  str(argument)+", " + str(mass_numb) + ")")
-            else:
+                warn("Redundant mass number information in isotope_symbol from"
+                     " inputs: (" + str(argument)+", " + str(mass_numb) + ")",
+                     UserWarning)
```

Comment:
Excuse me, do you have a moment to talk about our typing savior the f-string?
    
    f"Redundant mass number information in isotope_symbol from inputs: ({argument}, {mass_numb})",

Reply:
Oh, great idea!  I think I had some bad experiences using Fortran formatting (i.e., I used Fortran formatting) and so have been reluctant to learn how to do it in Python.